source/concurrent/Thread.ooc

```diff
@@ -64,9 +64,9 @@ version (unix || apple) {
 }
 
 version (windows) {
-	include windows
+	include inttypes
 
-	ThreadId: cover from DWORD {
+	ThreadId: cover from UInt {
```

Comment:
According to windows docs, DWORD is 32 bit type. For some reasons it was typedef'ed to UInt64, so I needed to change it to UInt.